pkg/network/infraconfigurators/bridge_test.go

```diff
@@ -0,0 +1,192 @@
+package infraconfigurators
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/vishvananda/netlink"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	netdriver "kubevirt.io/kubevirt/pkg/network/driver"
+)
+
+type Option func(handler *netdriver.MockNetworkHandler)
+
+func newMockedBridgeConfigurator(
+	vmi *v1.VirtualMachineInstance,
+	iface *v1.Interface,
+	handler *netdriver.MockNetworkHandler,
+	bridgeIfaceName string,
+	launcherPID int,
+	options ...Option) *BridgePodNetworkConfigurator {
+	configurator := NewBridgePodNetworkConfigurator(vmi, iface, bridgeIfaceName, launcherPID, handler)
+	for _, option := range options {
+		option(handler)
+	}
+	return configurator
+}
+
+var _ = Describe("Bridge infrastructure configurator", func() {
+	var (
+		ctrl    *gomock.Controller
+		handler *netdriver.MockNetworkHandler
+	)
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		handler = netdriver.NewMockNetworkHandler(ctrl)
+	})
+
+	AfterEach(func() {
+		ctrl.Finish()
+	})
+
+	const (
+		bridgeIfaceName = "br0"
+	)
+
+	newVMIWithBridgeInterface := func(namespace string, name string) *v1.VirtualMachineInstance {
+		vmi := v1.NewMinimalVMIWithNS(namespace, name)
+		vmi.Spec.Networks = []v1.Network{*v1.DefaultPodNetwork()}
+		vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{*v1.DefaultBridgeNetworkInterface()}
+		v1.SetObjectDefaults_VirtualMachineInstance(vmi)
+		return vmi
+	}
+
+	Context("discover link information", func() {
+		const (
+			ifaceName   = "eth0"
+			launcherPID = 1000
+		)
+
+		var (
+			bridgeConfigurator *BridgePodNetworkConfigurator
```

Comment:
Only one inner container uses this one, can you please move it there? (the others can use local ones)